### ISSUE TITLE
Forward Auth Search Box Redirect Issue

### ISSUE THREAD

**Issue by swesner411** [janeczku/calibre-web#2681]
**Describe the bug/problem**
While using Forward Auth, the search submit button routes to a local/internal IP address instead of the expected domain address. This causes the search function to break for remote users. If you manually enter a search URL, everything works as expected.

**To Reproduce**
1. Go to the Calibre-Web home page after passing through a Forward Auth service (i.e. Authentik)
2. Type something into the search box at the top of the page
3. Click submit
4. See browser address bar ( i.e. https://192.168.1.12:8083 )
5. Also see the Calibre-Web login screen at this point if you do not have a pre-existing session at the local address

**Logfile**
N/A

**Expected behavior**
Redirection to the right location ( i.e. https://my.company.com/search/stored/?query=test )

**Screenshots**
![Step 1](https://user-images.githubusercontent.com/23664751/216057970-545cf92a-a670-47fa-bd52-3fad1463a222.png)
![Step 2](https://user-images.githubusercontent.com/23664751/216057975-88476579-89aa-494e-85e8-8830dc3282a4.png)
![Step 3](https://user-images.githubusercontent.com/23664751/216060174-f7c095aa-0c79-455c-9924-58d1a3c691a2.png)

**Environment (please complete the following information):**
 - OS: Ubuntu Server 22.04.1 LTS
 - Python version: 3.10.6
 - Calibre-Web version: 0.6.19
 - Docker container: LinuxServer:
 - Special Hardware: N/A
 - Browser: Microsoft Edge Version 109.0.1518.70 & Chrome Version 109.0.5414.118

**Additional context**
Accessing via Nginx Proxy Manager -> Authentik -> Calibre-Web. All other functions and pieces of the Calibre-Web site are working as expected.

**Comment by swesner411** [comment]
I just tested the latest nightly build through the LinuxServer docker image and I am no longer experiencing this issue.

**Comment by OzzieIsaacs** [comment]
I didn‘t publish my change yet, so it was something different from  what I was thinking of

**Comment by swesner411** [comment]
Good to know. For your reference in case you want it, version reports as: 0.6.20 Beta - cf9a7d538fe39ea356138722f566dfd04bae8b54 - 2023-02-03T14:28:59+01:00

**Comment by swesner411** [comment]
> Please check the newest nightly again if it's still working, as I now published my changed code.

Just pulled the newest build and it's still working perfectly. Version reported is 0.6.20 Beta - 14d14637cda6be7d2a923cdd59efee1c906dbb80 - 2023-02-06T19:02:27+01:00

A bonus perk. The previous nightly wasn't loading ebooks at all for me for some reason, but this version is loading them up fine.

**Comment by OzzieIsaacs** [comment]
Yeah, the last version had the problem loading the epubs, which is now also fixed
